.circleci/config.yml

```diff
@@ -26,8 +26,34 @@ jobs:
             git update-index --assume-unchanged "Makefile"
 
       - run:
-          name: Validate Terraform format
-          command: make format-check
+          name: test-dependencies
+          command: |
+            #
+            # Install pre-commit
+            pip install pre-commit
+            #
+            # Install terraform
+            sudo apt-get install unzip
+            wget https://releases.hashicorp.com/terraform/${TERRAFORM_VER}/terraform_${TERRAFORM_VER}_linux_amd64.zip
+            unzip terraform_${TERRAFORM_VER}_linux_amd64.zip
+            sudo mv terraform /usr/local/bin/
+            terraform --version
+            #
+            # Install terraform-docs
+            sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+            curl -Lo ./terraform-docs https://github.com/terraform-docs/terraform-docs/releases/download/v0.10.1/terraform-docs-v0.10.1-$(uname | tr '[:upper:]' '[:lower:]')-amd64
+            chmod +x ./terraform-docs
+            sudo mv ./terraform-docs /usr/local/bin/terraform-docs
+
+      - run:
+          name: test1-terraform-format-and-docs-mod-a
```

Comment:
@exequielrafaela Just a couple of concerns regarding the names of these steps:
1- I would suggest not to use test1, test2, and so on. Perhaps just `test` is good enough to indicate the general purpose of the step.
2- I would also suggest not to use mod-a, mod-b, and so on. I think it would be more helpful to use either the name of the module under test or part of it, so that when we receive notifications we can tell which tests of which module failed.

Reply:
@diego-ojeda-binbash OJ agreed ✅ Thanks for your feedback :)